app/bundles/CoreBundle/Form/Type/EntityLookupType.php

```diff
@@ -108,6 +108,7 @@ public function configureOptions(OptionsResolver $resolver)
                             'class'              => "form-control {$options['model']}-select",
                             'data-chosen-lookup' => $options['ajax_lookup_action'],
                             'data-model'         => $options['model'],
+                            'main-entity'        => $options['main_entity'],
```

Comment:
no, I was trying to do a different approach to adding city name and country in the field name. forgot to remove it.